qiita_db/util.py

```diff
@@ -1520,17 +1499,25 @@ def generate_study_list(user, visibility):
                     info['owner'] = info['owner_email']
                 del info['owner_email']
 
-                # cleaning aids_with_deprecation
-                info['artifact_biom_ids'] = []
-                if info['aids_with_deprecation'] is not None:
-                    for x in info['aids_with_deprecation']:
-                        # f1-2 are the default names given by pgsql
-                        if not x['f2']:
-                            info['artifact_biom_ids'].append(x['f1'])
-                del info['aids_with_deprecation']
-
-                if info['preparation_data_types'] is None:
-                    info['preparation_data_types'] = []
```

Comment:
Is this no longer needed?

Reply:
This is correct, now the preparation data types is retrieved together with the artifact info, lines 1502 and 1511 below.